### ISSUE TITLE
Can't set password: Local admin account password is already set

### ISSUE THREAD

**Issue by kravciak** [rancher/rancher#33505]
**Rancher Server Setup**
- Rancher version: 4226d4ad0db1 (docker digest)
- Installation option Docker install:

**Describe the bug**
I am not able to set up local admin password on fresh rancher installation.
```
~ docker run -d --restart=unless-stopped -p 80:80 -p 443:443 --privileged rancher/rancher:master-head

It looks like this is your first time visting the Rancher UI,
but the local admin account password is already set to something unique.
Log in with that account below to continue the setup process.
```

**Screenshots**
![Screenshot from 2021-07-07 08-20-33](https://user-images.githubusercontent.com/538604/124713976-8092d580-df01-11eb-986d-f998062e641d.png)

**Comment by superseb** [comment]
This is because of the generated password on startup added in https://github.com/rancher/rancher/pull/32749, you can look up the password in the log or preset it using an environment variable.

**Comment by kravciak** [comment]
> ...you can look up the password in the log or preset it using an environment variable.

Could we improve the message here? Maybe a link to documentation ..or what you wrote is quite clear :)
This part of message gives no guidance on how to proceed:
```
..but the local admin account password is already set to something unique. Log in with that account below to continue the setup process.
```

**Comment by superseb** [comment]
I agree, this also seems in line with https://github.com/rancher/dashboard/pull/2870#issuecomment-831462681.

**Comment by kravciak** [comment]
Great, so it'a already in your scope, thanks for the links.
Should I keep this issue open for reference or close it?

**Comment by superseb** [comment]
I would transfer it to `rancher/dashboard` for review.
